Landing for AWS Marketplace

fiskus opened the pull request:
![image](https://user-images.githubusercontent.com/533229/211553744-23c7c3c9-8a2f-449d-ad51-ab9f97da36d6.png)

TODO:
* Links for "Read more"
* Links for One Liner (PDF) and Marketplace Listing
* Add more padding space for Case Studies block?
* Move Mailchimp endpoint and Calendly link somewhere?

- [ ] ~[Changelog](CHANGELOG.md) entry~ Probably, we should miss changelog entry here

nl0 commented on catalog/app/components/Form/TextField.tsx:
```diff
@@ -11,20 +11,26 @@ const useFieldInputStyles = M.makeStyles({
   ...rest
 }: FieldProps & M.TextFieldProps) {
   const inputClasses = useFieldInputStyles()
+  const readyForValidation = (input.value && meta.modified) || meta.submitFailed
```
does this mean the field would be ready for validation after a single letter is entered? doesn't seem like a good UX

fiskus replied:
I think there are some single letter names

fiskus replied:
final-form validates anyway
This check is just about showing validation error. So, better naming would be something like `readyToShowValidationError`

nl0 replied:
i meant that we probably shouldn't show validation errors until the form is submitted (we do this everywhere across the app), bc it's quite bad UX to show the errors right away and especially as you type (it feels quite annoying)

fiskus replied:
> we do this everywhere across the app
Except one place from where I copied that piece of code :)

Fixed, thanks

nl0 commented on catalog/app/website/pages/AwsMarketplace/CaseStudies.tsx:
```diff
@@ -0,0 +1,125 @@
+        <div className={classes.columns}>
+          <CaseStudy
+            title="Improving Data Management with Quilt and NextFlow"
+            content="Tessera Therapeutics is a pioneer in gene writing, using technology to insert messages into genomes to treat diseases. The company needed to manage and share more than 12 terabytes of scientific data across large teams of wet scientists and computational biologists. To address this, Tessera implemented Quilt Data and Nextflow to ensure that data was findable, accessible, inte ...
```
i'd put this into `children` for better readability/formatting

nl0 commented on catalog/app/website/pages/AwsMarketplace/Features.tsx:
```diff
@@ -0,0 +1,79 @@
+      <Feature
+        logo={dataVersionControl}
+        title="Dataset Version Control"
+        content="Quilt Data provides dataset immutability by allowing users to create and share datasets that are sealed and cryptographically verifiable, ensuring that the data cannot be altered after it has been shared. This helps to maintain the integrity and reliability of the data and ensures that users can trust the data they are working with."
```
same here re: `content` vs `children`

nl0 commented on catalog/app/website/pages/AwsMarketplace/Form.tsx:
```diff
@@ -0,0 +1,310 @@
+              variant="contained"
+              disabled={submitting || (submitFailed && hasValidationErrors)}
+            >
+              Submit
```
"submit" (and "tanks for submission") seems too abstract. we should use a more descriptive language here. tbh i don't understand what this form does at all. like, what am i "submitting"? subscribing to the mailing list? requesting a sales call? the UX is quite confusing

nl0 commented on catalog/app/website/pages/AwsMarketplace/Hero.tsx:
```diff
@@ -0,0 +1,121 @@
+        >
+          One Pager (PDF)
+          <M.Icon className={classes.linkIcon} color="inherit">
+            chevron_right
```
pls check out the ChevronLink component to make this a little bit more DRY (if it fits this use case)

fiskus replied:
Thanks, it fits

Do you mind, if I add hover effect like this?

![hover](https://user-images.githubusercontent.com/533229/211861178-ebce9a38-68cc-4721-bb4f-0c8e2723e329.gif)

nl0 replied:
sure, it looks nice

nl0 commented on catalog/app/website/pages/AwsMarketplace/index.tsx:
```diff
@@ -0,0 +1,58 @@
+  // (required for AwsMarketplace component)
+  return (
+    <Layout>
+      <MetaTitle />
```
no title segment for this specific page?

nl0 reviewed (commented):
looks ok. some nitpicks:

1. resilience logo has some strange blackish outline
2. call to action in the form is unclear and confusing
3. input style is inconsistent with all the other inputs we use (it's not MUI) -- maybe it's not such a big deal, since the website doesn't have any other inputs afair, but it doesn't seem right (and imo it conflicts a little bit with the button styling)

fiskus commented:
> input style inconsistent

I'm going to copy styles from search bar:
![Screenshot from 2023-01-11 22-27-55](https://user-images.githubusercontent.com/533229/211910948-b0ff2ed9-efca-48c2-8221-9969ed5e9f75.png)

fiskus commented:
![Screenshot from 2023-01-11 23-26-29](https://user-images.githubusercontent.com/533229/211921252-7cd38753-06c9-427d-a2ca-0c5f2759d004.png)
![Screenshot from 2023-01-11 23-26-05](https://user-images.githubusercontent.com/533229/211921271-d5903e1b-1a38-4ed2-9203-078ba50d6e46.png)

nl0 commented:
> ![Screenshot from 2023-01-11 23-26-29](https://user-images.githubusercontent.com/533229/211921252-7cd38753-06c9-427d-a2ca-0c5f2759d004.png) ![Screenshot from 2023-01-11 23-26-05](https://user-images.githubusercontent.com/533229/211921271-d5903e1b-1a38-4ed2-9203-078ba50d6e46.png)

this looks very nice

akarve commented on catalog/app/website/pages/AwsMarketplace/AwsPartner.tsx:
```diff
@@ -0,0 +1,56 @@
+      <div className={classes.content}>
+        <AwsPartner className={classes.awsPartner} />
+        <M.Typography variant="body1" color="textSecondary">
+          Quilt Data is an Amazon Advanced Technology Partner. Quilt brings seamless
```
```suggestion
          Quilt Data is an AWS Advanced Technology Partner. Quilt brings seamless
```

akarve reviewed (changes requested):
Regarding MailChimp form
1. Can we left-align (rather than center) text
2. Can we improve color? Some pale white with transparency would work.
3. "By clicking SUBMIT I agree to receive emails from Quilt"

Regarding Quilt + AWS
* Try to match letter sizes across logos so it's more balanced
* Use some transparency on "+"

akarve commented on catalog/app/website/pages/AwsMarketplace/Hero.tsx:
```diff
@@ -0,0 +1,108 @@
+          color="primary"
+          onClick={bookIntro}
+        >
+          Intro to customer
```
```suggestion
          Refer a customer
```

akarve commented:
Also:
Case study
1. logo sizes need to be balanced (either Celsius smaller or other bigger)
2. 

> 

I suggested "Refer a customer"

akarve commented:
> @QuiltSimon Can you find a clearer way to explain "Intro to Customer"? It seems hard to understand without context. It may be worth either making it a full-column button (or just link text), a la "Introduce a Customer to Us"

@drernie I suggested "Refer a customer"

akarve commented:
Please be sure we're matching font sizes with e.g. https://quiltdata.com/aws. @QuiltSimon there's too much text and it looks odd. Please trim if possible.

akarve commented on catalog/app/website/pages/AwsMarketplace/Features.tsx:
```diff
@@ -0,0 +1,89 @@
+        and reliability of the data and ensures that users can trust the data
+        they are working with.
+      </Feature>
+      <Feature heading="Data Commons" logo={dataCommons}>
```
```suggestion
      <Feature heading="Social Knowledge" logo={dataCommons}>
```
Data Commons is pretty loaded. You cold try "Data Mesh", idk.

akarve commented on catalog/app/website/pages/AwsMarketplace/Features.tsx:
```diff
@@ -0,0 +1,89 @@
+        data. The Quilt web catalog also allows users to search for datasets using
+        keywords, making it easy to find the data they need.
+      </Feature>
+      <Feature heading="Visualizations" logo={visualizations}>
```
```suggestion
      <Feature heading="Visualization" logo={visualizations}>
```

akarve commented on catalog/app/website/pages/AwsMarketplace/Features.tsx:
```diff
@@ -0,0 +1,89 @@
+  return (
+    <div className={cx(classes.root, className)}>
+      <Feature heading="Dataset Version Control" logo={dataVersionControl}>
+        Quilt Data provides dataset immutability by allowing users to create and share
+        datasets that are sealed and cryptographically verifiable, ensuring that the data
+        cannot be altered after it has been shared. This helps to maintain the integrity
+        and reliability of the data and ensures that users can trust the data
+        they are working with.
```
Go active voice and address subject directly with You! Text also gets A LOT shorter. @QuiltSimon 

```suggestion
        Create and share sealed, immutable datasets with cryptographic fingerprints
        that guarantee data are pristine and unchanged. Maintain data integrity, reliability,
        and lineage so that every dataset is trustworthy.
```

akarve commented on catalog/app/website/pages/AwsMarketplace/Features.tsx:
```diff
@@ -0,0 +1,89 @@
+  const classes = useStyles()
+  return (
+    <div className={cx(classes.root, className)}>
+      <Feature heading="Dataset Version Control" logo={dataVersionControl}>
```
USE ACTIVE VOICE
```suggestion
      <Feature heading="Verify" logo={dataVersionControl}>
```

fiskus commented:
* Left-aligned text in form
* Used transparent white background for the form
* "By clicking SUBMIT I agree to receive emails from Quilt"
* Made "Quilt" logo a little smaller, so it has the same height as "aws"
* Added transparency on "+"
* Made "Book a demo" button outlined
* Made some changes to font sizes so they are the same as on /aws page

![Screenshot from 2023-01-13 15-57-27](https://user-images.githubusercontent.com/533229/212336775-6466dec7-d1aa-42db-86e2-6f0fe9c428a7.png)

nl0 commented:
> * Left-aligned text in form
> 
>     * Used transparent white background for the form
> 
>     * "By clicking SUBMIT I agree to receive emails from Quilt"
> 
>     * Made "Quilt" logo a little smaller, so it has the same height as "aws"
> 
>     * Added transparency on "+"
> 
>     * Made "Book a demo" button outlined
> 
>     * Made some changes to font sizes so they are the same as on /aws page
> 
> 
> ![Screenshot from 2023-01-13 15-57-27](https://user-images.githubusercontent.com/533229/212336775-6466dec7-d1aa-42db-86e2-6f0fe9c428a7.png)

it's just a nitpick, but the spacing around "+" doesn't look right -- it's larger on the right than on the left

fiskus commented:
I changed the logos in figma, so "+" sign is more close to "Quilt" and "aws" now

QuiltSimon commented on catalog/app/website/pages/AwsMarketplace/CaseStudies.tsx:
```diff
@@ -0,0 +1,154 @@
+import * as M from '@material-ui/core'
+
+import logoCelsius from './logo-celsius.png'
+import logoResilience from './logo-resilience.png'
```
New Resilience Logo: 
![Resilience-1](https://user-images.githubusercontent.com/116831980/213199022-63e97bc2-f687-44bd-a6b1-6a61626c5700.jpeg)

fiskus replied:
I'll use this, if you don't mind https://resilience.com/wp-content/uploads/2021/11/Resilience-Logo.svg

QuiltSimon commented on catalog/app/website/pages/AwsMarketplace/Features.tsx:
```diff
@@ -0,0 +1,87 @@
+import visualizations from './visualizations.svg'
+
+const useFeatureStyles = M.makeStyles((t) => ({
+  root: {
```
Padding on this is good, but the behavior on shrink is odd -- my feeling is we should go from 3 across the 3 stacked for mobile. 

@akarve @robnewman @fiskus what are your thoughts on this? Effort to achieve? Certainly not a top-priority, but worth a call out.

fiskus replied:
> from 3 across the 3 stacked for mobile
yes and done

QuiltSimon commented on catalog/app/website/pages/AwsMarketplace/Form.tsx:
```diff
@@ -0,0 +1,347 @@
+
+import TextField, { TextFieldProps } from 'components/Form/TextField'
+import log from 'utils/Logging'
+import * as validators from 'utils/validators'
```
What are these validators doing? I was able to submit an email with no @ symbol. Not a top priority but worth consideration.

fiskus replied:
There are basic validators here, and the only one I use is "required" (requires user to fill input field)

For email validation there is another check https://github.com/quiltdata/quilt/pull/3251/files#diff-635522e4007d740d663650285e405e29132f1801c2b070667c213915981f51cfR179

It's quite simple, just checks `@` and `.`, and I was unable to submit form email without a `@`

QuiltSimon replied:
OK. Am I correct in understanding that this is now applied? (I'm still a relative PR noob, so I can't tell whether this is in the latest rev)

fiskus replied:
It was applied on 13 of January
I don't know, maybe you checked out the first revision only?

QuiltSimon commented on catalog/app/website/pages/AwsMarketplace/Hero.tsx:
```diff
@@ -0,0 +1,111 @@
+      marginLeft: t.spacing(1),
+    },
+  },
+  buttonOutlined: {
```
Hover behavior is beautiful, but this needs to be more prominent. Can it be semi-transparent?

QuiltSimon commented on catalog/app/website/pages/AwsMarketplace/Hero.tsx:
```diff
@@ -0,0 +1,111 @@
+          color="primary"
+          onClick={bookIntro}
+        >
+          Refer a customer
```
I'd like to change this to "Install Quilt", which links to our marketplace listing here: 
https://aws.amazon.com/marketplace/seller-profile?id=865bcbb9-ae88-4eed-8cfe-c58948910e53

fiskus commented:
* Changed Resilience logo, and added Resilience, Tessera and DSP Concepts to carousel
* Optimized layout for mobile view
* Changed call-to-action buttons. I think these three buttons should be different from each other, because they all in one viewport. So, "Install Quilt" becomes link button, and "Book a demo" - regular primary button

fiskus commented on catalog/app/website/pages/AwsMarketplace/Hero.tsx:
```diff
@@ -0,0 +1,111 @@
+        >
+          Refer a customer
+        </M.Button>
+        <M.Button
```
Buttons are in center, but text is left aligned (so on the left there is straight edge, but on the right some empty spaces in each line). That's why visually it looks like buttons are more on the right

I made width of buttons smaller (more standard) and now they look more centered

fiskus commented on catalog/app/website/pages/AwsMarketplace/Partners.tsx:
```diff
@@ -0,0 +1,60 @@
+
+import LogosCarousel from 'website/pages/Landing/LogosCarousel'
+
+import logoAllencell from 'website/pages/Landing/Logos/logo-allencell.png'
```
I copied DSP Concept logo (SVG) from their website, and made it grayscale

QuiltSimon reviewed (approved):
> I'll use this, if you don't mind https://resilience.com/wp-content/uploads/2021/11/Resilience-Logo.svg 

Just fine.

QuiltSimon commented:
> Regarding MailChimp form
> 
> 1. Can we left-align (rather than center) text
> 2. Can we improve color? Some pale white with transparency would work.
> 3. "By clicking SUBMIT I agree to receive emails from Quilt"
> 
> Regarding Quilt + AWS
> 
> * Try to match letter sizes across logos so it's more balanced
> * Use some transparency on "+"

I believe these are completed. Requesting rereview.

drernie commented:
Can someone show me which files contain the actual (new) marketing text for review? I'm getting bogged down...

QuiltSimon commented:
Features.tsx

Simon Kohnstamm

Director of Sales, Quilt Data, Inc.

***@***.*** | LinkedIn ( https://www.linkedin.com/in/simonrk/ )

(781) 277-2255

Manage Data Like Code

quiltdata.com ( https://quiltdata.com/ )

On Wed, Feb 1 2023 at 8:46 PM, Dr. Ernie Prabhakar < ***@***.*** > wrote:

> 
> 
> 
> 
> 
> 
> 
> Can someone show me which files contain the actual (new) marketing text
> for review? I'm getting bogged down...
> 
> 
> 
> —
> Reply to this email directly, view it on GitHub (
> https://github.com/quiltdata/quilt/pull/3251#issuecomment-1413032444 ) , or
> unsubscribe (
> https://github.com/notifications/unsubscribe-auth/A33LN3HWQIETTSQIVQ6PIZTWVMGVTANCNFSM6AAAAAATVUPESA
> ).
> You are receiving this because you were mentioned. Message ID: <quiltdata/quilt/pull/3251/c1413032444
> @ github. com>
> 
> 
>

drernie commented:
> Features.tsx

This is fine for now.  We will probably want to revisit all the Marketing pages once we agree on a new "Positioning Statement", but we shouldn't let that block getting this out for ISVa.
